### DISCUSSION TITLE
Both Feature-Policy and Permission-Policy are specified

### DISCUSSION THREAD

**Discussion by DavideWalder** [zitadel/zitadel#5087] [Q&A] [answered]
When working with iFrames, I get the following error from the browser (Chrome):
```
Error with Feature-Policy header: Some features are specified in both Feature-Policy and Permissions-Policy header: payment. Values defined in Permissions-Policy header will be used.
```

I noticed that both are specified: https://github.com/zitadel/zitadel/blob/632639ae7fe81db4fbabad6dd6928a39e4db0740/internal/api/http/middleware/security_headers.go#L76

I don't know enough about the topic to know if this is correct behaviour; was setting the same property in both intentional?

**Comment by livio-a** [comment, marked as answer]
Hey @DavideWalder 

By the time we implemented the policy, it was called Feature Policy. In the meantime, the name and syntax changed. While the Permission Policy is also still a draft of the W3C, we wanted to make sure the policy is applied by as many browsers as possible and therefore set both.
We will remove the Feature Policy header in the future. Until then, there shouldn't be any problem other than the log message.
